server/v1/tests/profile.test.js

```diff
@@ -21,3 +104,398 @@ describe("GET /api/v1/profile/test", () => {
       );
   });
 });
+
+/*
+ * @route   GET /api/v1/profile
+ * @desc    Return Current Users Profile
```

Comment:
Description doesn't match whats being done

Reply:
I left original description of route so developers know what it is supposed to do, and in the "describe" I describe how I'm not using as intended and it is expected to fail